docs/api/saved-objects/copy_to_space.asciidoc

```diff
@@ -0,0 +1,290 @@
+[role="xpack"]
+[[saved-objects-api-copy-to-space]]
+=== Copy to Space
+++++
+<titleabbrev>Copy to Space</titleabbrev>
+++++
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
```

Comment:
Is this still the correct way to mark an API as experimental?

Reply:
For the changes I've made, I kept this piece in tact. I believe Kibana is paving the way for how we communicate experimental functionality, so I say we keep it.